index.ts

```diff
@@ -1,31 +1,51 @@
 import { multiUrlGenerator } from './lib/multiUrlGenerator';
-import fetch from 'node-fetch';
+import fetch, { Response } from 'node-fetch';
+import console from 'console';
+import readline from 'readline';
+const URL_CONFIG_PATH = './config/url_components';
+const PARSER_PATH = './config/parsers/';
+type ScraperMode = 'wikipedia' | 'mojang';
+const MODES: ReadonlyArray<ScraperMode> = ['wikipedia', 'mojang'];
 
-// Test fetching Wiki articles
-const requestUrls = multiUrlGenerator(
-  'https://$lang.wikipedia.org/wiki/$article',
-  [
-    { $lang: 'en', $article: 'Plato' },
-    { $lang: 'es', $article: 'Aristóteles' },
-  ]
-);
+const client = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
 
-async function main() {
-  for (const url of requestUrls) {
+client.question('Choose the mode you want to run the program ', (MODE) => {
+  //TODO: change the question method to setPrompt()/popPromt() or reformulate the question() string to ask an actual question.
```

Comment:
I think it's ok not to ask an actual question here. If anything, we could prompt: "Enter the scraper mode name (suported modes: ${MODES.join(', ')})".

Reply:
Or alternatively, to make it a question: "What mode should I use? (available modes: ${MODES.join(', ')})"